packages/@o3r/telemetry/src/builders/index.ts

```diff
@@ -35,38 +34,36 @@ export const createBuilderWithMetrics: BuilderWrapper = (builderFn, sendData = d
       throw err;
     }
     finally {
-      try {
-        const endTime = Math.floor(performance.now());
-        const duration = endTime - startTime;
-        // context.builder.builderName does not contain the package name
-        const builderName = context.builder.name as string;
-        context.logger.info(`${builderName} run in ${duration}ms`);
-        const environment = getEnvironmentInfo();
-        const data: BuilderMetricData = {
-          environment,
-          duration,
-          builder: {
-            name: builderName,
-            options,
-            ...(context.target
-              ? {
-                target: {
-                  name: context.target.target,
-                  projectName: context.target.project,
-                  configuration: context.target.configuration
-                }
-              } : {}
-            )
-          },
-          error
-        };
-        context.logger.debug(JSON.stringify(data, null, 2));
-        await sendData(data, context.logger);
-      } catch (e: any) {
+      const endTime = Math.floor(performance.now());
+      const duration = endTime - startTime;
+      // context.builder.builderName does not contain the package name
+      const builderName = context.builder.name as string;
+      context.logger.info(`${builderName} run in ${duration}ms`);
+      const environment = getEnvironmentInfo();
+      const data: BuilderMetricData = {
+        environment,
+        duration,
+        builder: {
+          name: builderName,
+          options,
+          ...(context.target
+            ? {
+              target: {
+                name: context.target.target,
+                projectName: context.target.project,
+                configuration: context.target.configuration
+              }
+            } : {}
+          )
+        },
+        error
+      };
+      context.logger.debug(JSON.stringify(data, null, 2));
```

Comment:
we should probably remove this

Reply:
It was more for us when we run the command with `--verbose`